package.json

```diff
@@ -65,6 +65,8 @@
 		"npm": "^10"
 	},
 	"dependencies": {
+		"@ant-design/icons": "^5.6.1",
+		"antd": "^5.23.4",
```

Comment:
Should these packages be in web/package.json?

I can see in CI you have the error

```
#17 0.986 error during build:
#17 0.986 [vite]: Rollup failed to resolve import "@ant-design/icons" from "/home/node/web/src/pages/ConfirmationPage.jsx".
#17 0.986 This is most likely unintended because it can break your application at runtime.
```

Which mat possibly be resolved by moving these dependencies?